vignettes/analysing-n-grams.Rmd

```diff
@@ -74,7 +74,7 @@ from the newly created file.^[When reading this data without `guess_max = 2000`,
 a warning is raised because the column type was not recognized properly for one
 column. Increasing `guess_max` 
 helped in this instance.]
-
+(#2)
```

Comment:
This footnote is a bit unclear to me. stay in the same tone and tense please: either you are informing, or you are reporting. in this case it should be informing e.g. "raising g_max can help".